iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -172,7 +171,8 @@ public enum ErrorCode
         ModuleAlreadyExistsOnDevice = 409301,
 
         /// <summary>
-        /// The etag in the request does not match the etag of the existing resource, as per <see href="https://datatracker.ietf.org/doc/html/rfc7232">RFC7232</see>. The etag is controlled by the service and is based on the device identity it should not be updated in normal operations.
+        /// The etag in the request does not match the etag of the existing resource, as per <see href="https://datatracker.ietf.org/doc/html/rfc7232">RFC7232</see>.
+        /// The etag is controlled by the service and is based on the device identity it should not be updated in normal operations.
```

Comment:
new line

Reply:
"The etag is controlled by the service and is based on the device identity it should not be updated in normal operations." - was this meant to mean "...is based on the device identity _and_ it should not be ~updated~ used in normal operations."?

Reply:
I'd add something to the effect of "In order to get the up-to-date ETag for a twin use <link to registryManager.getTwin(deviceId)>"

Reply:
And maybe mention that this is a mechanism for protecting against the race conditions of multiple clients updating the same resource and overwriting each other. Linking the RFC is nice, but I don't expect most people to want to follow the link in order to learn why we are doing this